### ISSUE TITLE
Kafka tools running on Kafka pods print entire GC log

### ISSUE THREAD

**Issue by ppatierno** [strimzi/strimzi-kafka-operator#1172]
Currently on master because the GC logging is enabled by default if you run a Kafka tool on one of the Kafka pod, you get an entire GC log before the results.

```
sh-4.2$ bin/kafka-topics.sh --list --zookeeper localhost:2181
OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
2018-12-17T12:29:52.141+0000: 0.407: [GC pause (G1 Evacuation Pause) (young), 0.0091134 secs]
   [Parallel Time: 8.5 ms, GC Workers: 1]
      [GC Worker Start (ms):  407.3]
      [Ext Root Scanning (ms):  1.9]
      [Update RS (ms):  0.0]
         [Processed Buffers:  0]
      [Scan RS (ms):  0.0]
      [Code Root Scanning (ms):  0.2]
      [Object Copy (ms):  6.2]
      [Termination (ms):  0.0]
         [Termination Attempts:  1]
      [GC Worker Other (ms):  0.0]
      [GC Worker Total (ms):  8.3]
      [GC Worker End (ms):  415.6]
   [Code Root Fixup: 0.0 ms]
   [Code Root Purge: 0.0 ms]
   [Clear CT: 0.0 ms]
   [Other: 0.6 ms]
      [Choose CSet: 0.0 ms]
      [Ref Proc: 0.3 ms]
....
      [GC Worker Total (ms):  5.7]
      [GC Worker End (ms):  1462.0]
   [Code Root Fixup: 0.0 ms]
   [Code Root Purge: 0.0 ms]
   [Clear CT: 0.0 ms]
   [Other: 1.6 ms]
      [Choose CSet: 0.0 ms]
      [Ref Proc: 0.4 ms]
      [Ref Enq: 0.0 ms]
      [Redirty Cards: 0.0 ms]
      [Humongous Register: 0.1 ms]
      [Humongous Reclaim: 0.0 ms]
      [Free CSet: 0.0 ms]
   [Eden: 5120.0K(5120.0K)->0.0B(6144.0K) Survivors: 1024.0K->1024.0K Heap: 14.4M(128.0M)->10.5M(128.0M)]
 [Times: user=0.00 sys=0.00, real=0.01 secs]
my-topic
test
Heap
 garbage-first heap   total 131072K, used 10752K [0x00000005cb000000, 0x00000005cb100400, 0x00000007c0000000)
  region size 1024K, 2 young (2048K), 1 survivors (1024K)
 Metaspace       used 17165K, capacity 17574K, committed 17792K, reserved 1064960K
  class space    used 2261K, capacity 2350K, committed 2432K, reserved 1048576K
```

it's because we are setting the `KAFKA_GC_LOG_OPTS` which is used by the tools as well.

We should try to decouple tools behavior from what we configured on the brokers.
As suggested by @scholzj we should use a different env var to set in the cluster operator and then "translating" it to the `KAFKA_GC_LOG_OPTS` (recognized by Kafka) just into the run script for the broker. In this way the tools won't see it.

**Comment by scholzj** [comment]
@ppatierno I assume this issue can be closed now after #1176 was merged?

**Comment by ppatierno** [comment]
Yes! Closing.
